blocklylib-core/src/main/java/com/google/blockly/utils/ColorUtils.java

```diff
@@ -83,21 +83,21 @@ public static int parseColor(@NonNull String str, @Nullable float[] tempHsvArray
         char firstChar = str.charAt(0);
         if (firstChar == '#' && str.length() == 7) {
             try {
-                result = Integer.parseInt(str.substring(1, 7), 16);
-            } catch (NumberFormatException e) {
+                return Color.parseColor(str);
+            } catch (IllegalArgumentException e) {
                 throw new ParseException("Invalid hex color: " + str, 0);
             }
-            return result;
         } else if (Character.isDigit(firstChar) && str.length() <= 3) {
             try {
                 int hue = Integer.parseInt(str);
-                result = getBlockColorForHue(hue, tempHsvArray);
+                return getBlockColorForHue(hue, tempHsvArray);
             } catch (NumberFormatException e) {
                 throw new ParseException("Invalid color hue: " + str, 0);
             }
+        } else {
```

Comment:
remove return result; from comment.

But this brings up a good point - do you think anyone could be depending on the return null in this case vs. the exception? If not, LGTM after removing the return from the comment.

Reply:
There is a very slim chance someone might. The documentation explicitly says a ParseException is thrown, so they really shouldn't have any such expectation.

Fixed the comment.